boot/bootdata/bootcd.ini

```diff
@@ -1,19 +1,33 @@
 [FREELOADER]
-DefaultOS=Setup
-TimeOut=0
+DefaultOS=LiveCD_Debug
+TimeOut=10
 
 [Display]
-TitleText=ReactOS Setup
+TitleText=ReactOS Setup & Live BootCD
 MinimalUI=Yes
 
 [Operating Systems]
-Setup="Setup"
+Setup="ReactOS Setup (Text Mode)"
 Setup_Debug="Setup (Debug)"
 Setup_Aacpi="Setup ACPI APIC (Debug)"
 Setup_Screen="Setup (Screen)"
+;;;;
+LiveCD="ReactOS Setup (Graphics Mode) & Live"
+LiveCD_Debug="ReactOS Live (Debug)"
+LiveCD_Macpi="LiveCD ACPI SMP (Debug)"
+LiveCD_Aacpi="LiveCD ACPI APIC (Debug)"
+LiveCD_VBoxDebug="ReactOS Live (VBox Debug)" 
+LiveCD_Screen="ReactOS Live (Screen)"
+; LiveCD_LogFile="ReactOS Live (Log file)"
+LiveCD_RamDisk="ReactOS Setup (Graphics Mode) & Live in RAM"
```

Comment:
You used here in several locations "ReactOS Setup (Graphics Mode)" vs. "ReactOS Setup (Text Mode)". You did that only for some, but not for all options. E.g. I do have no clue which kind of setup I will get for "Setup (Debug)", "Setup ACPI APIC (Debug)", "Setup (Screen)". I do assume the txt-mode. Most likely you avoided the duplication because it would otherwise become relatively confusing to read all those multiple options and word-combinations at once. I do wonder whether it would be easier for the user to anticipate what he will get on each selection if we used something slightly shorter, which is **only one word to differentiate this aspect** instead of two, but that shorter thing everywhere, so the user knows for each selection what it is: e.g. just "GUI" vs. "TXT".

Reply:
Yes, forget about this file. This is part of the first temporary commit of this PR that was there in order to have something bootable and testable. The actual phrasing is not optimal, indeed. And I would like to have freeldr support sub-menus so that we don't have "ReactOS Setup" (for example) repeated over and over for each boot entry, but just one, and the sub-menu items are the different boot debug options for example.